adage/analyze/tests.py

```diff
@@ -432,25 +448,25 @@ def call_get_API(self, uri):
         '''
         Helper method: asserts that GET method is allowed via input URI.
         '''
-        resp = self.api_client.get(uri, data={'format': 'json'})
```

Comment:
Correct. I thought we had previously gotten rid of all the `format=json`s in our `api_client` calls, but these ones slipped by until I saw this calling method.